### PR TITLE
misc changes while translating (smoe:Docs29 misc pc 1)

### PR DESCRIPTION
**Pull request by smoe**
Series of smallish changes - (almost no) semantic line breaks.

### CODE REVIEW

**Review by hansu — CHANGES_REQUESTED** [review]

File: docs/src/gcode/g-code.adoc [hunk 1]
```diff
@@ -2567,102 +2565,90 @@ becomes the specified value. All of the coordinate system's origins
...
+Do not use `G92.1` in the remainder of the first program.
+The parameter values will be saved when the first program exits and restored when the second one starts up.
+Use `G92.3` near the beginning of the second program.
+That will restore the offsets saved in the first program.
 
 [[gcode:g93-g94-g95]]
 == G93, G94, G95 Feed Rate Mode(((G93, G94, G95 Feed Rate Mode)))
 
-* 'G93' - is Inverse Time Mode.
+* `G93` - is Inverse Time Mode.
```
[the remark this answers is not in the document]
Comment (smoe) [reply docs/src/gcode/g-code.adoc:2623]: This likely was inconsistent prior to me editing already. I like the idea that G93 is a command that should be printed in monospace. I likely eventually observed that inconsistency myself and then stopped and am at ease with whatever decision you make, just do not want to see those patches delayed. Let's just decide how it shall be and we then have more dedicated patches triggered by that decision.

File: docs/src/remap/remap.adoc [hunk 2]
```diff
@@ -382,7 +369,7 @@ all of them make sense. The following combinations are useful idioms:
 `python=`__<pythonfunction>__ `modalgroup=`__<group>__::
   Directly call to a Python function without any argument conversion.
   The most powerful way of remapping a code and going straight to
-  Python. Use this if you don't need an NGC procedure, or NGC is
+  Python. Use this if you don_t need an NGC procedure, or NGC is
```
[the remark this answers is not in the document]
Comment (smoe) [reply docs/src/remap/remap.adoc]: done

**Comment by smoe** [comment]
@hansu - I do not know about what requested change there remains unanswered.

**Comment by smoe** [comment]
@hansu - rebased, also squashed.

**Comment by hansu** [comment]
@smoe Again conflicts to resolve...

**Comment by smoe** [comment]
@hansu - conflicts resolved and building

**Comment by hansu** [comment]
The thing with the monospaced highlighting especially for the G-codes. 
The problem (at least for me) is that the monospaced font has different proportions than the regular one. So with the same font size, the characters are smaller (height). So a bit unsuitable if a highlighted character is smaller than the rest.
But if I increase the font size, the monospace text is then wider than the regular one. Looks also weird. For single chars like "G1" it looks good, bot not for longer text:
grafik

I have no really good solution for that. The easiest fix for highlighting would be for
- only a few chars: italic 
- longer code names: monospaced

**Comment by smoe** [comment]
I have no immediate answer.  Important for me is to distinguish anything that is coded from the text that task about it. If the semantics are correct then I can live with most optical representations as long as they consistent.

That said, it may be wrong to highlight a command as `typed` just when expressing that the command is implemented very nicely. Maybe it would be sufficient to just have the cross-link on it?

But then, I also liked how we have now all the pin names identified, even when just talking about them. For me this is helpful.

What is it that we really want? Using LaTeX with a function that knows how to present the pin names/commands?

**Comment by silopolis** [comment]
Can't the CSS be adjusted with a smaller size for mono fonts?
Or use better quality web fonts?

**Comment by hansu** [comment]
> Can't the CSS be adjusted with a smaller size for mono fonts?

The example shows an enlarged monospaced font. With a smaller size te character hight is lower than the regular font.

 >Or use better quality web fonts?
No idea what font can be used which every browser supports. Or we provide an own font.

**Comment by silopolis** [comment]
Le sam. 7 janv. 2023 à 19:02, Hans Unzner ***@***.***> a
écrit :

> Can't the CSS be adjusted with a smaller size for mono fonts?
>
> The example shows an enlarged monospaced font. With a smaller size te
> character hight is lower than the regular font.
>

Oh didn't understood sorry

Or use better quality web fonts?
> No idea what font can be used which every browser supports. Or we provide
> an own font.
>
>
Aren't web fonts downloaded by browser on demand?

**Comment by hansu** [comment]
>Aren't web fonts downloaded by browser on demand?

I think so. But for offline docs we should provide the font.

**Comment by hansu** [comment]
I was so free and did a clean rebase (and add a commit on top)

**Pull request merged**